Eevee.Sleep.Bot/Handlers/ButtonClickedHandler.cs

```diff
@@ -0,0 +1,35 @@
+using Discord.WebSocket;
+using Eevee.Sleep.Bot.Controllers.Mongo;
+using Eevee.Sleep.Bot.Models;
+
+namespace Eevee.Sleep.Bot.Handlers;
+
+public static class ButtonClickedHandler {
+    public static async Task DisplayRoleButtonClicked(
+        ButtonInteractionInfo info,
+        SocketGuildUser user
+    ) {
+        var roleToRemove = DiscordRoleContoller
+            .FindAllTrackedRoles()
+            .Select(r => r.Id)
+            .Intersect(user.Roles.Select(r => r.Id))
+            .ToArray();
+
+        await user.RemoveRolesAsync(roleToRemove);
+        await user.AddRoleAsync(info.CustomId);
+    }
+
+    public static async Task AddRoleButtonClicked(
+        ButtonInteractionInfo info,
+        SocketGuildUser user
+    ) {
+        await user.AddRoleAsync(info.CustomId);
+    }
+
+    public static async Task RemoveRoleButtonClicked(
```

Comment:
Since this is a single `Task`, no need `async Task`, just return the `Task` of `user.AddRoleAsync()`. 

https://stackoverflow.com/a/38017167/11571888